src/auth-service/utils/user.util.js

```
@@ -3737,10 +3712,217 @@
+                        )
+                    ),
+                  };
+                }
+                return groupRole.role;
+              })()
+            : groupRole.role,
+        }));
+      }
+
+      // 3. Populate network roles
+      if (userObj.network_roles && userObj.network_roles.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)